lib/orders/calculate_fees.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @param {RESTv2} rest
+ * @param {Order} order
+ * @param {Object} orderParams
+ * @returns {Promise<void>}
+ */
+module.exports = async (rest, order, orderParams) => {
+  const { symbol, submittedAt } = orderParams
+  const trades = await rest.accountTrades(symbol, submittedAt)
+  const tradeForOrder = trades.find(t => t.orderID === order.id)
+
+  if (!tradeForOrder) {
+    return null
+  }
+
+  const {
+    execAmount,
+    execPrice,
+    maker: isMaker,
+    fee: amount,
+    feeCurrency: currency
+  } = tradeForOrder
+
+  const cost = Math.abs(amount * execPrice)
```

Comment:
since fees are mostly in lower float values .. better to use `BigNumber` or `toPrecision` here to prevent floating point issue?